lib/timely/time_field.rb

```diff
@@ -0,0 +1,20 @@
+module Timely
+  module TimeField
+    def self.included(base)
+      base.extend(ClassMethods)
+    end
+
+    module ClassMethods
+      def time_fields(*attributes)
+        attributes.each do |attribute|
+          method_name = if attribute.to_s.ends_with?('time')
+            attribute.to_s + '_of_day'
+          else
+            attribute.to_s + '_time_of_day'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.